### PR TITLE
Add linting and unit tests execution to the Travic-CI

### LINKED ISSUE #20: Get Initial Travis Integration Setup
## What happens?

Travis CI does not build a push container images to quay.io yet.

## What were you expecting to happen?

Travis CI builds code, builds container images, and pushes to quay.io.

## Steps to reproduce: 

n/a

## Any errors, stacktrace, logs? 

n/a

## Environment:

* Runtime version(Java, Go, Python, etc): n/a
* Desktop OS/version: n/a

## Additional comments:

none

### PR DESCRIPTION
**Pull request by bjaroszewski**
Signed-off-by: bjaroszewski <bjaroszewski@griddynamics.com>

# Pull Request Template

## Description

Add unit tests run and golint

Fixes #20

## Type of change

* Bug fix (non-breaking change which fixes an issue)

## Environment

* Runtime version(Java, Go, Python, etc):

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,11 @@ env:
 
 before_install:
   - travis_retry go mod vendor
+  - go get golang.org/x/lint/golint
+
+script:
+  - golint $(go list ./... | grep -v /vendor/)
```
Comment (seanmalloy) [inline .travis.yml]: Missing the `-set_exit_status` command line flag.

This command should be `golint -set_exit_status $(go list ./... | grep -v /vendor/)`

**Comment by seanmalloy** [comment]
Looks like we just need to fix some golint errors ...

```
/home/travis/gopath/src/github.com/KohlsTechnology/eunomia/test/init.go:32:2: a blank import should be only in a main or test package, or have a comment justifying it
/home/travis/gopath/src/github.com/KohlsTechnology/eunomia/test/init.go:36:1: exported function Initialize should have comment or be unexported
/home/travis/gopath/src/github.com/KohlsTechnology/eunomia/test/init.go:70:1: exported function AddToFrameworkSchemeForTests should have comment or be unexported
/home/travis/gopath/src/github.com/KohlsTechnology/eunomia/test/e2e/util.go:30:1: exported function WaitForPod should have comment or be unexported
/home/travis/gopath/src/github.com/KohlsTechnology/eunomia/version/version.go:20:2: exported var Version should have comment or be unexported
Found 5 lint suggestions; failing.
The command "golint -set_exit_status $(go list ./... | grep -v /vendor/)" exited with 1.
```

**Comment by Smiley73** [comment]
Yeah and we should probably go through Sonar also and get that cleaned up. If I remember correctly, it was calling out a lot of that.

**Comment by seanmalloy** [comment]
We will get `golint` cleaned up first. We will worry about Sonar later.

File: .travis.yml [hunk 2]
```diff
@@ -8,6 +8,11 @@ env:
 
 before_install:
   - travis_retry go mod vendor
+  - go get golang.org/x/lint/golint
+
+script:
+  - golint -set_exit_status $(go list ./... | grep -v /vendor/)
```
Comment (seanmalloy) [inline .travis.yml]: I looks like we are missing two commands here ...

```
gofmt -d .
go vet $(go list ./... | grep -v /vendor/)
```

**Pull request merged**